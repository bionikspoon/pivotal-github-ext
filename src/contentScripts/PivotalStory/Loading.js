/* @flow */
import React from 'react'
import LoadingSpinner from '../LoadingSpinner'
import Well from '../Well'
import './PivotalStory.css'

export default function Loading() {
  return (
    <Well className="PivotalStory__container">
      <LoadingSpinner />
    </Well>
  )
}
