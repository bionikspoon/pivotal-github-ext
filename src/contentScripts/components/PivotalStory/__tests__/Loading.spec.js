import React from 'react'
import { shallow } from 'enzyme'
import Loading from '../Loading'

describe('components/Loading', () => {
  test('it renders correctly', () => {
    const props = {}
    const wrapper = shallow(<Loading {...props} />)

    expect(wrapper).toMatchSnapshot()
  })
})
