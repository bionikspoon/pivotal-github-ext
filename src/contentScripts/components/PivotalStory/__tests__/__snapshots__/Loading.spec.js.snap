// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Loading it renders correctly 1`] = `
<Well
  className="PivotalStory__container"
>
  <LoadingSpinner />
</Well>
`;
